### DISCUSSION TITLE
How many letters does the Gaelic alphabet have?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#674] [Galaxy Trivia] [answered]
*answer in binary*

**Comment by peterbaccaro** [comment]
Scottish Gaelic uses the same alphabet (A, B, C …) as English, though Gaelic uses just 18 of the 26 letters so in binary this would be **10010**.

**Comment by SANKARAMDAS** [comment, marked as answer]
Today people write and type Irish Gaelic with the standard Latin alphabet. The Irish alphabet uses 24 of the 26 letters of the English alphabet, as opposed to the original number of 18. That said, there are few words with j, k, v, x, y or z, and the ones which do exist are generally words incorporated from English. Examples include zú (zoo), vóta (vote), yóyó (yo-yo).
And 18 Binary number (5 digits) is 10010.
Binary signed 2's complement (16 digits) - 0000000000010010
